docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -23,21 +25,107 @@ If the tested page does not use HTTPS-specific features (like
 [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API),
 [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation_API),
 [ApplePaySession](https://developer.apple.com/documentation/apple_pay_on_the_web/applepaysession), or
-[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)), TestCafe successfully proxies it over HTTP. Otherwise, the tests fail because the page throws JavaScript errors, does not load or loads partially. This indicates that you should [configure the proxy server to use HTTPS](#proxy-the-website-over-https).
+[SubtleCrypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto)), TestCafe successfully proxies it over HTTP. Otherwise, the tests fail because the page throws JavaScript errors, does not load or loads partially.
 
-### Proxy the Website Over HTTPS
+To establish a secure HTTPS-connection, Node.js requires a valid [PKCS12](https://en.wikipedia.org/wiki/PKCS_12) certificate (`.pfx`). If you have a certificate signed by a trusted authority, we recommend that you [use this certificate](#use-a-trusted-certificate) if your security policy permits. The browsers are then able to open the tested page without workarounds. Alternatively, you can [generate and use a self-signed certificate](#use-a-self-signed-certificate).
```

Comment:
- "If you have a certificate signed by a trusted authority, we recommend that you [use this certificate](#use-a-trusted-certificate) if your security policy permits" Why do you recommend something and then add a condition to it?

- "are then able to" Do you mean 'can'?

Reply:
- I wanted to introduce a clickable descriptive link ("we recommend that you **use this certificate**"). If we write "we recommend that you **use it**" i feel like it'll be harder to notice. Please advise

- Yes. I used "able to" to avoid repetition, there is a 'can' in the following sentence

Reply:
I would just say: We recommend that you [a certificate signed by a trusted authority](#use-a-trusted-certificate). This allows a browser to open the tested page without workarounds. You can also [generate and use a self-signed certificate](#use-a-self-signed-certificate).

OR

You can [generate and use a self-signed certificate](#use-a-self-signed-certificate), but we recommend that you [a certificate signed by a trusted authority](#use-a-trusted-certificate). This allows a browser to open the tested page without workarounds.